bciers/apps/reporting/src/app/components/operations/OperationReview.tsx

```diff
@@ -0,0 +1,161 @@
+  operationReviewUiSchema,
+} from "@reporting/src/app/utils/jsonSchema/operations";
+
+export default function OperationReview({
+  formData,
+  version_id,
+}: {
+  formData: any;
+  version_id: UUID;
+}) {
+  const customStepNames = [
+    "Operation Information",
+    "Facilities Information",
+    "Compliance Summary",
+    "Sign-off & Submit",
+  ];
+
+  return (
+    <Box sx={{ p: 3 }}>
+      <div className="container mx-auto p-4">
+        <MultiStepHeader step={0} steps={customStepNames} />
+      </div>
+      <Grid container spacing={3}>
+        <Grid item xs={12} md={3}>
+          <Paper elevation={3} sx={{ p: 2 }}>
+            <Typography variant="h6" gutterBottom>
+              Operation Information
+            </Typography>
+            <List component="nav">
+              <ListItem button selected>
+                <ListItemIcon>
+                  <CheckCircleOutline htmlColor="green" />
+                </ListItemIcon>
+                <ListItemText
+                  primary="Review operation information"
+                  primaryTypographyProps={{ color: "#1A5A96" }}
+                />
+              </ListItem>
+              <ListItem button>
+                <ListItemText primary="Review facilities" />
+              </ListItem>
+              <ListItem button>
+                <ListItemText primary="Person responsible" />
+              </ListItem>
+            </List>
+          </Paper>
+        </Grid>
+        <Grid item xs={12} md={9}>
+          <Typography variant="h6" color="#38598A" fontSize="24px" gutterBottom>
+            Review operation information
+          </Typography>
+          <Paper sx={{ p: 2, mb: 3, bgcolor: "#DCE9F6", color: "#313132" }}>
+            <Box sx={{ display: "flex", alignItems: "center" }}>
+              <InfoIcon sx={{ mr: 1 }} />
+              <Typography variant="body2">
+                The information shown on this page is data entered in
+                registration. You can edit it here but it will only apply to
+                this report. To make edits for all of your reports, please edit
+                this information in registration.
+              </Typography>
+            </Box>
+          </Paper>
+          <Typography
+            variant="body2"
+            color="#38598A"
+            fontSize="16px"
+            gutterBottom
+          >
+            Please ensure this information was accurate for Dec 31 2024
+          </Typography>
+          <FormBase
+            schema={operationReviewSchema}
+            uiSchema={operationReviewUiSchema}
+            formData={formData}
+            onSubmit={async (data: { formData?: any }) => {
+              const method = "POST";
+              const endpoint = `reporting/save-report/${version_id}`;
+              const pathToRevalidate = `reporting/save-report/${version_id}`;
+              const formDataObject = JSON.parse(JSON.stringify(data.formData));
+              const body = {
```

Comment:
We should be able to pass the formData as the request body directly if we name the same thing in the json schema. 
Any reason why we need to provide a default for the fields?

Reply:
I just had a default value for when it was missing in the passed data so I could easily tell whats going on. I could remove the defaults now